src/logger.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { format } from 'node:util';
+import Interfaces from '@oclif/core/interfaces';
+import { Logger } from '@salesforce/core';
+
```

Comment:
Logger is a top-level export now
```suggestion
import { Logger } from '@salesforce/core/logger';
```